app/forms/provider_interface/offer_wizard.rb

```diff
@@ -59,7 +59,7 @@ def conditions_to_render
     end
 
     def course_option
-      CourseOption.find(course_option_id)
+      CourseOption.find_by(id: course_option_id)
```

Comment:
in order to avoid `ActiveRecord::RecordNotFound`? what are the circumstances in which a record wouldn't be found, and would we want to do anything about it?

Reply:
This happens when we've changed course but not selected a course option yet.